### PR TITLE
Aeyb #5 user profile page

### LINKED ISSUE #5: Profile Page for Each User
### Outcome

As a board member, I can view my profile page on the webapp, update my availability status, access my stats, and view my roles in the organization

### Moment

Functioning Profile Page with Availability status and Roles view

### Key Results
- [x] Get profile picture from Google Account
- [x] Get user's full name from Google Account
- [ ] User can change their availability status
- [x] User can view their assigned roles
- [x] User can access their Meeting statistics page
- [x] User can access their Personal statistics page
- [x] Code the desired design from the designer's wireframe
- [x] Code checked by another developer
- [x] Create pull request

### PR DESCRIPTION
**Pull request by long715**
PR for the current draft of the Profile Page! Feel free to leave any feedback/suggestions :))

### CODE REVIEW

File: frontend/src/pages/profiles/ProfileInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    // updates the status during onclick events
+    const [status, setStatus] = React.useState<UserStatus>(UserStatus.Idle);
+    // makes the nsub navigation for setStatus visibile during an on click event
+    const [editClick, setEdit] = React.useState(false);
+
+    return (
+        <body>
+            <div id="title">
+                <h1>Profile</h1>
+            </div>
```
Comment (pumbas600) [inline frontend/src/pages/profiles/ProfileInfo.tsx]: An HTML page should only ever have one `<body>` tag, which is already in the index HTML file. A normal `<div>` should work just as well though :)

**Comment by pumbas600** [comment]
Ooooo, this is looking really nice Violet :))

Two small thing is I've noticed is:
1.  You use ids in a lot of places. I personally would recommend class names instead as they work just as well even if it's only for one thing and it means the component can be reused without violating the uniqueness of the id (Which would probably just make it complain rather than throw any errors '^^).

2. I noticed that you use the import `@reacticons/ionicons` in `ProfileInfo.tsx` but don't seem to have installed it in the package.json. Is it there locally for you? (In which case you just need to add that to your next commit), otherwise, you may have accidentally installed it globally, in which case you can just do `npm i @reacticons/ionicons` while in the frontend folder to update the package.json file :)

**Comment by long715** [comment]
currently in need of reviewing from other devices, works fine on my desktop. 
- using "sticky" for the navigation bar (kind of buggy when u change window size)
- need to update to @TylerDonghwi navigation bar design 
- may have to update responsiveness for bigger screens

pls comment any feedback, would be appreciated!

**Comment by hillaro** [comment]
> currently in need of reviewing from other devices, works fine on my desktop.

You can use Chrome's built-in Inspect tool to check the responsiveness of the webpage without pulling out other devices!
image
If the start bar doesn't show up for you, press Ctrl + Shift + M, or you can toggle device toolbar 
image

**Review by luke-mccallum — APPROVED** [review]
looks great

**Pull request merged**